run.c

```diff
@@ -71,6 +71,9 @@ void run(void)
 
         /*Convert back to floating point time*/
         set_global_time(exp(loga_from_ti(All.Ti_Current)));
```

Comment:
What if set_global_time takes the integer time instead? Then there is no rounding off error.

Reply:
This is a little tricky (because pesky globals). We must do set_global_time before read_snapshot, because we need All.cf.a, to get the velocity units when reading the snapshot. But before we use an integer time we must do init_timebins. And before we do that we need to do setup_sync_points. I *think* there is no need for setup_sync_points to be after set_global_time...so yes, we can probably do that.

Reply:
Except we must do set_global_time(1.0) in sfr_eff.c, and 1.0 may not be on the integer timeline. (also *before* we have init_timebins). That code wants cleaning up at some point...but not today

Reply:
But I can do set_global_time(exp(loga_from_ti(All.Ti_Current))) instead of All.TimeInit anyway, which should get rid of the roundoff.